components/lib/Divider/Divider.module.scss

```diff
@@ -1,5 +1,6 @@
 .divider {
   &--horisontal {
+    margin: 1rem 0 0;
```

Comment:
do NOT add margin for all dividers. Add margin by my-2 className where Divider is used